src/Frame.cpp

```diff
@@ -25,6 +25,7 @@ const size_t Frame::COORDSIZE_ = 3 * sizeof(double);
 // ---------- CONSTRUCTION/DESTRUCTION/ASSIGNMENT ------------------------------
 /// CONSTRUCTOR
 Frame::Frame( ) : 
+  OnlyFreeV_(false),
```

Comment:
good name.

I know from now, I should always come to your desk and ask a name. lol.